shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/ShivaVariantcalling.scala

```diff
@@ -48,9 +49,22 @@ class ShivaVariantcalling(val parent: Configurable) extends QScript
 
   var inputBqsrFiles: Map[String, File] = Map()
 
+  var genders: Map[String, Gender.Value] = _
+
   /** Executed before script */
   def init(): Unit = {
     if (inputBamsArg.nonEmpty) inputBams = BamUtils.sampleBamMap(inputBamsArg)
+    if (genders == null) genders = {
```

Comment:
This can probably better be an Option, with a default value of `None`, in stead of checking for null